### PR TITLE
Increase the Interval value for send_health_check_pass check

### PR DESCRIPTION
**Pull request by Gsantomaggio**
- Add  [* 500 on send_health_check_pass ](https://github.com/rabbitmq/rabbitmq-peer-discovery-consul/pull/5/files#diff-69cf72a92713fa96e3d8bb4df1e84405R68)
The default value called the check too often here:
```
Starting Consul health check notifier (effective interval: 30 milliseconds)
consul     |     2017/11/15 13:34:51 [DEBUG] agent: Node info in sync
consul     |     2017/11/15 13:34:51 [DEBUG] http: Request PUT /v1/agent/check/pass/service%3Arabbitmq%3A1c77f97939fe (263.6µs) from=172.30.0.3:44621
``` 

now it is:

```
Starting Consul health check notifier (effective interval: 15000 milliseconds)
```
 
- Add docker compose example with consul

- Uploaded the Docker image [3.7-consul-BETA]( https://hub.docker.com/r/pivotalrabbitmq/rabbitmq-autocluster/tags/)

### CODE REVIEW

File: src/rabbit_peer_discovery_consul.erl [hunk 1]
```diff
@@ -551,7 +551,7 @@ start_session_ttl_updater(SessionId) ->
     M = ?CONFIG_MODULE:config_map(?BACKEND_CONFIG_KEY),
     Interval = get_config_key(consul_svc_ttl, M),
     rabbit_log:debug("Starting session renewal"),
-    {ok, TRef} = timer:apply_interval(Interval * 500, ?MODULE,
```
Comment (michaelklishin) [inline src/rabbit_peer_discovery_consul.erl]: The value we have is consistent with https://github.com/rabbitmq/rabbitmq-autocluster/blob/stable/src/autocluster_consul.erl#L132.

Comment (Gsantomaggio) [reply src/rabbit_peer_discovery_consul.erl]: yes, I am changing the value, I got where is the problem!

**Comment by Gsantomaggio** [comment]
@michaelklishin now it is ok, I set the same value for the interval check as here:
https://github.com/rabbitmq/rabbitmq-autocluster/blob/stable/src/autocluster_consul.erl#L135

The default interval value [was 30 milliseconds](https://github.com/rabbitmq/rabbitmq-peer-discovery-consul/blob/rabbitmq-server-1257/src/rabbitmq_peer_discovery_consul_health_check_helper.erl#L67)

**Comment by michaelklishin** [comment]
Thank you!

**Pull request merged**